src/patient-lab-dashboard/patient-lab-details/patient-lab-details.test.tsx

```diff
@@ -30,20 +30,18 @@ import {
   mockBahmniEncounterErrorResponse,
   mockBahmniEncounterRequest,
   mockBahmniEncounterResponse,
-  mockEncounterTypeResponse,
+  mockEncounterTypeResponse
 } from '../../__mocks__/encounter.mock'
 import {mockPendingLabOrder} from '../../__mocks__/patientLabDetails.mock'
 import {mockPendingLabOrdersResponse} from '../../__mocks__/pendingLabOrders.mock'
 import {
   mockEmptyReportTableResponse,
-  mockReportTableResponse,
+  mockReportTableResponse
 } from '../../__mocks__/reportTable.mock'
 import {
-  diagnosticReportRequestBodyWithBasedOn,
-  mockDiagnosticReportErrorResponse,
-  mockDiagnosticReportResponse,
-  mockLabTestsResponse,
-  mockUploadFileResponse,
+  diagnosticReportRequestBodyWithBasedOn, mockDiagnosticReportErrorResponse,
```

Comment:
There is no need to reformat this. It is easier to read and modify to have one import per line

Reply:
changes have been made